<launch>
  <arg name="device" default="/dev/ttyUSB0"/>
  <arg name="baud_rate" default="230400"/>

  <node pkg="nodelet"
    type="nodelet"
    name="kr_serial_interface"
    args="standalone kr_serial_interface/QuadSerialComm"
    output="screen">
    <param name="device" value="$(arg device)"/>
    <param name="baud_rate" value="$(arg baud_rate)"/>
    <remap from="~to_robot" to="to_robot"/>
    <remap from="~from_robot" to="from_robot"/>
  </node>

  <node pkg="nodelet"
    type="nodelet"
    name="quad_decode_msg"
    args="standalone kr_serial_interface/QuadDecodeMsg"
    output="screen">
    <remap from="~serial" to="from_robot"/>
  </node>
</launch>
